app/infrastructure/router.go

```diff
@@ -67,6 +70,10 @@ func init() {
 		authorized.POST("/users", func(c *gin.Context) { userController.StoreUser(c) })
 		authorized.POST("/users/csv", func(c *gin.Context) { userController.StoreUsers(c) })
 
+		//Point endpoints
+		authorized.GET("/points", func(c *gin.Context) { userController.TopPointsUsers(c) })
+		authorized.GET("/rewards/:employeeId", func(c *gin.Context) { pointsController.PointsByEmployeeId(c) })
```

Comment:
feel like we should keept it the same..
/points to get everything
/points/:employee ID to get their history..
hcanging the name seems a bit weird

Reply:
We can't create `/users/points`. Because `/user/:employeeId` affect. If you want to create `/users/points`. We should change it to query params like`?id=<employeeId>`.